src/main/java/com/textmessenger/model/Message.java

```diff
@@ -30,5 +36,7 @@
   private String message;
 
   @Column(name = "created_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  @CreatedDate
```

Comment:
Don't use java.utils.Date,use java.time.LocalDateTime instead

Reply:
there are some reliable sources?

Reply:
It's okay to use both